core/domain/platform_feature_services.py

```diff
@@ -33,23 +33,30 @@
 from __future__ import annotations
 
 from core import feconf
-from core import platform_feature_list
+from core import platform_feature_list as feature_list
```

Comment:
Try not to alias, just use the full name of the module. (If the module name is wrong then rename the module.)

Reply:
Got it, Updated.
Thanks